### DISCUSSION TITLE
GitHub app installation: The selected plan has been retired.

### DISCUSSION THREAD

**Discussion by coderbyheart** [renovatebot/renovate#22869] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Describe the problem

When trying to install the app in a new repository, I get this error:

> The selected plan has been retired. To "Complete order and begin installation", you need to select an active plan from the applications listing page.

![image](https://github.com/renovatebot/renovate/assets/188915/1dbe7c0b-c8fd-4875-b4f2-f7edb3237dd9)

![Screenshot_20230619_111404](https://github.com/renovatebot/renovate/assets/188915/7403ee3e-d7dd-44fe-8711-376246ca2b63)

**Comment by rarkins** [comment, marked as answer]
You should not see retired plans - seems like a GitHub bug. I see this:

![image](https://github.com/renovatebot/renovate/assets/6311784/cf429e72-f4ed-4a26-8db1-2c9b70511843)

However it's not important, see https://github.com/renovatebot/renovate/discussions/22856#discussioncomment-6209731

**Reply by coderbyheart** [reply]
Alright, thanks!
